third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -951,6 +1002,14 @@ struct AsyncCopyGlobalToLocalOpConversion
     }
 
     auto moduleOp = op->getParentOfType<ModuleOp>();
+    auto freeVarMasks = getFreeVariableMasks(srcTy);
+    // NOTE(@peterbell10): We must load redundant data on different CTAs, since
+    // they don't access the same shared memory.
```

Comment:
Suggestion:

```
// NOTE(@peterbell10): We must load redundant data across different CTAs.
// Otherwise, we'd have to access data in shared memory to broadcast 
// the same loaded values to multiple addresses.
```

Reply:
Changed to:
```cpp
// NOTE(@peterbell10): We load redundant data on different CTAs, so the data
// is available in each CTAs respective shared memory. Otherwise, we would
// need an additional broadcast step to copy the data between CTAs.
```